File: javascript/stack.js
const prompt = require('prompt-sync')();

let stack = [1, 2, 3];

console.log('length', stack.length)

push = (data) => {
    stack.push(data);
}

pop = () => {
    stack.pop();
}

const operation = prompt('choose operation 1->push | 2->pop | 3->quit');
console.log(operation == 1);

if (operation == 1) {
    const data = prompt('enter element to be pushed');
    push(data);
    prompt(`stack is ${stack}`)
} else if (operation == 2) {
    pop();
    prompt(`stack is ${stack}`)
}

File: python/ds/stack.py
stack = [1]


def push():
    element = input("enter ele to be pushed")
    stack.append(element)
    print(stack, '| ', type(stack))


def pop():
    stack.pop()
    print(stack, '| ', type(stack))


while True:
    choice = int(input("choose operation 1->push | 2->pop | 3->quit"))

    if choice == 1:
        push()
    elif choice == 2:
        pop()
    elif choice == 3:
        break
    else:
        print("enter any of the given operation")
